icalevents/icalparser.py

```diff
@@ -336,7 +355,7 @@ def parse_events(content, start=None, end=None, default_span=timedelta(days=7)):
                         start_tz = timezones[str(e.start.tzinfo)]
                     else:
                         try:
-                            start_tz = timezone(str(e.start.tzinfo))
+                            start_tz = e.start.tzinfo
```

Comment:
I'm not sure what the try-except was originally suppose to catch but I suspect it's related to the `timezone(str(...))` logic, if that's the case we should probably remove them, same for L366-369. Does it break any tests if you remove it?

Reply:
good catch! :heart: